.github/ISSUE_TEMPLATE/01-bug.yml

```diff
@@ -0,0 +1,123 @@
+---
+name: Bug Report
+description: File a bug report.
+title: "[Bug] "
+labels: ["bug", "triage-needed"]
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Thanks for taking the time to fill out this bug report!
+  - type: dropdown
+    id: dag-factory-version
+    attributes:
+      label: DAG Factory Version
+      description: What version of DAG Factory are you running? If you do not see your version in the list, please (ideally) test on
+        the latest release or main to see if the issue is fixed before reporting it.
+      options:
+        - "0.19.0"
+        - "main (development)"
+        - "Other DAG Factory version (please specify below)"
```

Comment:
We should probably add an example but not hard-coded versions. It probably does not make sense for us to have to update this every release we do - it will become stale unless we find a way to automate this.

Reply:
yes, make sense totally! Removed the dropdown and we now have a free text input to accept the version.